fix(ingestion/looker): fix doc for sql parsing documentation (#10883)
datahub-project/datahub: metadata-ingestion/docs/sources/looker/lookml_post.md

#### Configuration Notes

> 1.  If a view contains a liquid template (e.g. `sql_table_name: {{ user_attributes['db']}}.kafka_streaming.events, with db=ANALYTICS_PROD`), then you will need to specify the values of those variables in the `liquid_variable` config as shown below 

Reviewer: ```suggestion
1. If a view contains a liquid template (e.g. `sql_table_name: {{ user_attributes['db']}}.kafka_streaming.events }}`, with `db=ANALYTICS_PROD`), then you will need to specify the values of those variables in the `liquid_variable` config as shown below:
```

>     ```

Reviewer: yml

>     ```

Reviewer: ```suggestion
    ```yml
```

      liquid_variable:
        user_attributes:
          db: ANALYTICS_PROD

> 

Reviewer: ```suggestion
```

#### Configuration Notes

> 1.  The sql parsing is disabled by default, but can be enabled by setting `parse_table_names_from_sql: True`.

Reviewer: should we enable this by default?

Author: It is done as part of https://github.com/datahub-project/datahub/pull/10892

Reviewer: we can remove this line then right

> 2.  If a view contains a liquid template (e.g. `{{ user_attributes['whatever'] }}`), then you will need to specify the values of those variables in the `liquid_variable` config

Reviewer: actually provide an example lookml code and corresponding recipe

also make it a bit more realistic - people might have something like this `sql_table_name: {{ user_attributes['db']}}.kafka_streaming.events`, with `db=ANALYTICS_PROD`

> 2.  If view contains liquid template then we can set liquid variable as key value pair in `liquid_variable` config parameter where key act as variable-name and value as variable's value

Reviewer: link to the looker docs on Liquid templates and provide an example of some lkml and the corresponding `liquid_variable` config

```suggestion
2.  If a view contains a liquid template (e.g. `{{ user_attributes['whatever'] }}`), then you will need to specify the values of those variables in the `liquid_variable` config
```
